python/init_container.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+if [ -n "${DCOS_CLI_AUTO_INIT}" ]; then
```

Comment:
```suggestion
if [[ -z ${SKIP_DCOS_CLI_INIT:-} ]]; then
```
This way we can omit the variable by default and only pass in the single place where it's needed.

Reply:
The initial idea was skip dcos-cli initialization by default and turn on (via DCOS_CLI_AUTO_INIT="yes") when it necessary. In that case, employing DCOS_CLI_AUTO_INIT look more clear from code semantic and UX perspective.

Reply:
I see. I'm not sure this achieves the clarity though. First, the word `AUTO` in the name seems mysterious to me, especially if the default is to *not* do it. Second, if we need to explicitly turn it on in all but one case (unless I missed something) then perhaps it's a sign it should be on by default after all? WDYT?

Reply:
> @porridge: if we need to explicitly turn it on in all but one case (unless I missed something) then perhaps it's a sign it should be on by default after all

Agreed. The default case should be a user executing the script, not a CI run.

Reply:
Okay, I refactored this stuff. PTAL